docs/Milestones.md

```diff
@@ -0,0 +1,8 @@
+ ## Descripción de Milestones y PMV
+
+# PMV - Milestone 0: Modelo base del proyecto
```

Comment:
¿Por qué modelo "base"? ¿Hay otro modelo encima de ese? ¿Por qué usáis palabras sin que tengan mucha relación con el contexto?